### PR TITLE
Add 'My Questions' and 'Starred Questions' to Forum

### PR DESCRIPTION
**Pull request by tmartin1**

### CODE REVIEW

**Comment by tmartin1** [comment]
Don't merge yet, fixing bug in comment voting.

**Comment by tmartin1** [comment]
Alright, everything should work as expected now.

File: client/app/forum/view/view.controller.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        $scope.currentQuestion.rating--;
+      } else {
+        $scope.user.forum.starred[qid] = true;
+        $scope.currentQuestion.rating++;
+      }
+      $http.put('/api/questions/' + qid, { rating: $scope.currentQuestion.rating });
+      $http.put('/api/users/stars', { starred: $scope.user.forum.starred });
+    }
+  };
+
```
Comment (scottrice10) [inline client/app/forum/view/view.controller.js:43]: Is upundefined and downundefined the best way to do this?

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**